editoast/src/schema/operation/mod.rs

```diff
@@ -36,6 +36,25 @@ pub enum CacheOperation {
     Delete(ObjectRef),
 }
 
+impl CacheOperation {
+    pub fn try_from_operation(
+        operation: &Operation,
+        railjson_object: RailjsonObject,
+    ) -> Result<Self> {
+        let cache_operation = match operation {
+            Operation::Create(railjson_object) => {
+                CacheOperation::Create(ObjectCache::from(railjson_object.deref().clone()))
+            }
+            Operation::Update(UpdateOperation { railjson_patch, .. }) => {
+                let railjson_object = railjson_object.patch(railjson_patch)?;
+                CacheOperation::Update(ObjectCache::from(railjson_object))
+            }
+            Operation::Delete(_) => CacheOperation::Delete(railjson_object.get_ref()),
```

Comment:
Same as Create, we should probably use the ObjectRef:
```suggestion
            Operation::Delete(op) => CacheOperation::Delete(op.into()),
```

Reply:
Done in 656d0e0f013acef528cce712cc6201105045d8b0.